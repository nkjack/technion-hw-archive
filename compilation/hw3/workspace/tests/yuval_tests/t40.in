void main(){
  int x;
}/* error */