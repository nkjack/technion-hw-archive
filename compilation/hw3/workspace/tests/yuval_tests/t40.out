---end scope---
main - 0 preconditions
x INT 0
line 3: syntax error
